lomond/session.py

```diff
@@ -245,7 +245,7 @@ def run(self, poll=5, ping_rate=30, auto_pong=True):
             yield events.Disconnected(graceful=True)
 
 
-if __name__ == "__main__":
+if __name__ == "__main__":  # pragma: no cover
```

Comment:
As we now have full tests, perhaps this should be removed instead of excluded.

If not, perhaps it would go well in an `examples` module, or similar.